content/id/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -0,0 +1,226 @@
+---
+reviewers:
+- girikuncoro
+title: Kontainer Sementara
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" for_k8s_version="v1.16" >}}
+
+Halaman ini memberikan gambaran umum tentang kontainer sementara: atu jenis 
+kontainer khusus yang berjalan sementara pada {{< glossary_tooltip term_id="pod" >}} 
+yang sudah ada untuk melakukan tindakan yang diinisiasi oleh pengguna seperti 
+dalam pemecahan masalah. Anda menggunakan kontainer sementara untuk memeriksa 
```

Comment:
our convention is to use "kamu" instead of "Anda" since it's more welcoming and friendly
```suggestion
dalam pemecahan masalah. Kamu menggunakan kontainer sementara untuk memeriksa 
```